tests/python/test_continous_dynamics.py

```diff
@@ -89,6 +89,140 @@ def test_multibody_free():
         pass
 
 
+def test_centroidal():
+    try:
```

Comment:
You don't need the try/except here. It's set in the previous test because pinocchio is an optional dependency